marketplace/mux-video/README.md

```diff
@@ -0,0 +1,86 @@
+![Contentful + Mux](https://banner.mux.dev/Contentful.svg)
+
+# Contentful Video Streaming Plugin
+
+A Contentful UI extension that makes it simple to add beautiful streaming via [Mux](https://https://mux.com) to your [Contentful](https://contentful.com) project. Just install the extension, add the component to your content model, and you're good to go! 🙌🏾
+
+Before getting started, make sure to generate a new [Access Token](https://dashboard.mux.com/settings/access-tokens). To make installation a little easier later, you can save the generated environment file as `.env` in this directory. If you're going to install the plugin [via the interface](#install-via-the-contentful-interface), just keep the token ID and secret handy.
+
+## Install via the CLI
+
+Make sure you have the Contentful CLI installed:
+
+```
+npm install contentful-cli -g
+```
+
+You are logged in:
+
+```
+contentful login
+```
+
+And have a space selected:
+
+```
+contentful space use
+```
+
+To install the UI Extension:
+
+```
+npm run install-extension
```

Comment:
Ah right, I cleaned up some of the scripts from our `examples` repo when I realized this was going to be in the marketplace with click-to-install rather than needing to be installed via the CLI.

Before I go in and just remove those lines from the Readme, what do you think? I assume it's superfluous given it'd be a rare installation path, but just in case... Here's what `install-extension` currently does in our examples repo:

```
"install-extension": "eval `cat .env` && contentful extension create --installation-parameters '{\"muxAccessTokenId\": \"'$MUX_TOKEN_ID'\", \"muxAccessTokenSecret\": \"'$MUX_TOKEN_SECRET'\"}'"
```